docs/integrations/containers-orchestration/activemq.md

```diff
@@ -67,7 +66,7 @@ Create the following Fields in Sumo Logic prior to configuring collection. This
 
 <TabItem value="k8s">
 
-If you're using ActiveMQ in a Kubernetes environment, create the fields:
+If you're using ActiveMQ in a Kubernetes environment, then these fields will be created:
```

Comment:
We need to get this out of tab view. And give a clear idea that fields created in non-K8S will be created always and pod_label_* fields will get created with we are dealing with K8S